client/src/utils/create-types.ts

```diff
@@ -2,10 +2,18 @@ type CreateTypesType = {
   [action: string]: string;
 };
 
-export function createTypes(
-  types: string[] = [],
-  ns = 'annon'
-): CreateTypesType {
+/**
+ * Creates an object in which the `keys` represent the action names and the
+ * `values` the action type.
+ * {
+ *   action: actionType,
+ *   ...
+ * }
+ * @param {array} types Names of the actions.
+ * @param {string ns Name of the namespace.
```

Comment:
```suggestion
 * @param {string} ns Name of the namespace.
```
I'm not that familiar with JSDoc, but this looks like it should be closed.